app/controllers/spotlight/mastheads_controller.rb

```diff
@@ -0,0 +1,16 @@
+module Spotlight
+  # Handles requests to upload images for exhibit and site masthead images
+  class MastheadsController < FeaturedImagesController
```

Comment:
Same question here wrt the possibility of simply using the `FeaturedImagesController`.